Firestore/Example/Podfile

```diff
@@ -10,6 +10,7 @@ target 'Firestore_Example' do
   pod 'FirebaseAuth', :path => '../../'
   pod 'FirebaseCore', :path => '../../'
   pod 'FirebaseFirestore', :path => '../../'
+  pod 'nanopb'
```

Comment:
This shouldn't be required because nanopb should be a dependency of FirebaseFirestore in (FirebaseFirestore.podspec).

The difference is that this is defining the local project while the podspec defines the dependencies you'll get when you pod install Firebase/Firestore.

Reply:
Done. (You're quite right; this isn't required.)